scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -1,31 +1,41 @@
 version 1.0
 
+import "GvsCreateTables.wdl" as CreateBQTables
+
 workflow GvsAssignIds {
 
   input {
     Array[String] external_sample_names
     String project_id
     String dataset_name
     String sample_info_table = "sample_info"
-    String sample_info_schema = "sample_name:STRING,sample_id:INTEGER,is_loaded:BOOLEAN"
-    String? service_account_json
+    String workspace_namespace
+    String workspace_name
+    String? service_account_json_path
     Int? preemptible_tries
     File? gatk_override
     String? docker
+    String uuid=""
+    String sample_info_schema_json = '[{"name": "sample_name","type": "STRING","mode": "REQUIRED"},{"name": "sample_id","type": "INTEGER","mode": "NULLABLE"}]'
```

Comment:
This seems duplicated between here and the GvsCreateTables WDL, can we just have it in one place (suggesting the GvsCreateTables WDL)